charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```
@@ -155,3 +155,10 @@ ingress:
   #  - secretName: chart-example-tls
   #    hosts:
   #      - oidc-discovery.example.org
+
+tests:
+  busybox:
+    image:
+      registry: docker.io
+      repository: library/busybox
+      version: latest
```

Chainguard images are known to be secured and smaller then others.

I like the fact they are close to 0 or 0 CVEs, they include signatures, provenance, and sboms.

https://github.com/chainguard-images/images/tree/main/images/busybox

```suggestion
      registry: cgr.dev
      repository: chainguard/busybox
      version: glibc-1.36
```

---

+1 for chainguard. can we use `latest`  instead of a specific version?  So we dont have to keep updating it.

---

We could, although it is a bad practice to use latest tags in general.

---

For an appversion i wouldnt use latest. For a busybox container to use in a test i think its ok.

---

I'm not against changing things to use chainguard, but there are other busyboxes in the chart other then the ones I had to modify here. Lets have a different pr do the chainguard stuff.